webpack.config.js

```diff
@@ -11,6 +11,13 @@ module.exports = {
         filename: "[name].js",
         path: path.resolve(__dirname, "_site/assets/js")
     },
+    cache: {
+        type: "filesystem",
```

Comment:
https://webpack.js.org/configuration/cache/#cachecompression

Shall we also try to add `brotli` compression since we have the type for cache as `file system`

Reply:
In development mode?

Reply:
`production` mode uses `gzip` compression by default. I believe webpack defaults are good.

Reply:
In general we can use the combination of gzip + brotli to serve files efficiently. Brotli saves us more space. 

> The advantage for Brotli over gzip is that it makes use of a dictionary and thus it only needs to send keys instead of full keywords. According to [certsimple](https://certsimple.com/blog/nginx-brotli),
> 
> Javascript files compressed with Brotli are 14% smaller than gzip.
> HTML files are 21% smaller than gzip.
> CSS files are 17% smaller than gzip.

https://tech.oyorooms.com/how-brotli-compression-gave-us-37-latency-improvement-14d41e50fee4

Reply:
I think the goal here is to help local development rather than deploy compressed assets, so I think the Webpack defaults are fine.